modules/openapi-generator/src/main/resources/php/README.mustache

```diff
@@ -4,29 +4,19 @@
 {{{appDescriptionWithNewLines}}}
 {{/appDescriptionWithNewLines}}
 
-This PHP package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
```

Comment:
Personally, I prefer putting this block on top for growth hacking but I'm ok to give your suggestion a try to move it to the bottom of the page.

Reply:
@wing328 as discussed in #7530, the "growth hacking" effort is possible without it being overwhelming.

 From the POV of the:
1. **generator developer**, the generator is the most important thing, the code generated is a side-effect, just test files.
2. **generator user**, the point of the generator is to generate the best possible package, not promote the generator itself, perpetuum mobile style. :)
3. **package user** (the developer using the generated code), the generator is totally irrelevant, it might as well not exist, what's important is the code offered in the package, how to use it and get stuff done.

POV in 1. and 3. are in total opposition, this change is trying to shift the focus a bit more toward 3. to better realize 2. The goals of 1. should still be realized, but in a more subtle way.

Reply:
BTW this same issue is visible [here](https://github.com/flexolabs/reddit-client/blob/25feb73a9df32154d14598c298726c0cb3d017bb/composer.json#L13): this URL is **not** the homepage of the package built, it's either the [`info.contact.url` provided in the spec](https://github.com/flexolabs/openapi-specs/blob/4718374fd1df2cbcef94529e6e5c01a02bb31989/reddit/v1/index.yaml#L9) or something the user provides.

Reply:
> the generator is totally irrelevant, it might as well not exist, what's important is the code offered in the package, how to use it and get stuff done.

Many users still not aware that the API clients can be auto-generated instead of manually written. It doesn't hurt to make them aware there's such thing called OpenAPI Generator that they can use to generate code.

Reply:
@wing328 I understand that as do I understand your desire to promote the generator, it's perfectly understandable and not unreasonable.

My intention here is to point out **generator's users have no inherent interest to promote the generator**, they're not its core developer team. What they want from the generator is generating high-quality code which they can offer as "SDKs" for their API's users (or use themselves). The generator for group 2. is a tool, a means to an end, it's not the destination.

The group 3. doesn't care about the generator at all, they're primarily interested to use the code generated, they're twice removed from it, for them, the code generated is the product. The code and the docs generated should reflect that better, aligning them a bit more toward the interest of users of the code generated goes a long way.

It's not about "informing the users of the package about the generator which made the package", it's about the package itself. The meta-documentation obviously has its place, but it's shouldn't be the primary focus.

Reply:
Think about it like this: if this was a printing press, it would be trying to add a chapter about itself to each book it's printing because it's trying to inform the readers about itself. :) Yes, it's in the printing press's best interest to put the word out, but there's more than one way to do it, being too direct can seem obnoxious and have the opposite effect from the desired one.

Reply:
I never said the promotional message should be the focus. I'm not against generating a better package. Putting the message at the top of the package doesn't mean we (openapi-generator contributors) are not focusing on the quality of the package.